cfme/tests/v2v/test_infrastructure_mapping.py

```diff
@@ -0,0 +1,46 @@
+"""Test to validate End-to-End migrations- functional testing."""
+import pytest
+
+from cfme.fixtures.provider import rhel7_minimal
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_VERSION, ONE_PER_TYPE
+
+pytestmark = [
+    pytest.mark.provider(
+        classes=[RHEVMProvider], selector=ONE_PER_VERSION, required_flags=["v2v"]
+    ),
+    pytest.mark.provider(
+        classes=[VMwareProvider],
+        selector=ONE_PER_TYPE,
+        fixture_name="second_provider",
+        required_flags=["v2v"],
+    ),
+]
+
+
+@pytest.mark.parametrize(
+    "form_data_vm_obj_single_datastore", [["nfs", "nfs", rhel7_minimal]], indirect=True
```

Comment:
since the fixture needs to know what is the type of Source and target storage provider. It could be 
`'nfs','nfs` 
`'nfs','iscsi'`
or any combo of those two.